android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/service/AdyenRepository.java

```diff
@@ -0,0 +1,46 @@
+package com.appcoins.sdk.billing.service;
+
+import com.adyen.checkout.base.model.PaymentMethodsApiResponse;
+import com.appcoins.sdk.billing.listeners.AdyenLoadPaymentInfoListener;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class AdyenRepository {
+
+  private Service bdsService;
+
+  public AdyenRepository(Service bdsService) {
+
+    this.bdsService = bdsService;
+  }
+
+  public void loadPaymentInfo(String method, String value, String currency, String walletAddress,
+      final AdyenLoadPaymentInfoListener listener) {
+    Map<String, String> queries = new LinkedHashMap<>();
+    queries.put("wallet.address", walletAddress);
```

Comment:
do this keys of this map change?shouldn maybe have a url builder more in context of our project that knows this keys?